File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/PersonPensjonsbeholdning.java
package no.nav.domain.pensjon.regler.grunnlag;

import no.nav.domain.pensjon.regler.Pakkseddel;

import java.io.Serializable;

public class PersonPensjonsbeholdning implements Serializable {


    private Pensjonsbeholdning pensjonsbeholdning;
    private Pakkseddel pakkseddel;
    private String fodselsnummer;

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/grunnlag/PersonPensjonsbeholdning.kt
package no.nav.pensjon.regler.domain.grunnlag

import no.nav.pensjon.regler.domain.Pakkseddel
import java.io.Serializable

class PersonPensjonsbeholdning : Serializable {
    var pensjonsbeholdning: Pensjonsbeholdning? = null
    var pakkseddel: Pakkseddel? = null
    var fodselsnummer: String? = null
}